<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultWidget</class>
 <widget class="QWidget" name="ResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="progressWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>24</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widgetNavigationButtons" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="buttonGoFirst">
            <property name="toolTip">
             <string>الصفحة الاولى</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../share/alkotobiya/qt_rc.qrc">
              <normaloff>:/menu/images/go-last.png</normaloff>:/menu/images/go-last.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>27</width>
              <height>24</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonGoPrev">
            <property name="toolTip">
             <string>الصفحة السابقة</string>
            </property>
            <property name="text">
             <string>..</string>
            </property>
            <property name="icon">
             <iconset resource="../share/alkotobiya/qt_rc.qrc">
              <normaloff>:/menu/images/go-next.png</normaloff>:/menu/images/go-next.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>24</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>101</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelNav">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="buttonGoNext">
            <property name="toolTip">
             <string>الصفحة التالية</string>
            </property>
            <property name="text">
             <string>التالي</string>
            </property>
            <property name="icon">
             <iconset resource="../share/alkotobiya/qt_rc.qrc">
              <normaloff>:/menu/images/go-previous.png</normaloff>:/menu/images/go-previous.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>24</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonGoLast">
            <property name="toolTip">
             <string>الصفحة الاخيرة</string>
            </property>
            <property name="text">
             <string>..</string>
            </property>
            <property name="icon">
             <iconset resource="../share/alkotobiya/qt_rc.qrc">
              <normaloff>:/menu/images/go-first.png</normaloff>:/menu/images/go-first.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>24</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../share/alkotobiya/qt_rc.qrc"/>
 </resources>
 <connections/>
</ui>
